java/jsch/Portfile

```diff
@@ -19,12 +20,14 @@ homepage                http://www.jcraft.com/jsch/
 master_sites            sourceforge:project/jsch/jsch/${version}/
 use_zip                 yes
 
-checksums               md5     0b9312909fe542f6e662d40ec676f6b5 \
-                        sha1    c4252657c7caad9c8482836ddb1fa2abcf080357 \
-                        rmd160  cc0d72d817829c434dfcba845d2ff0d43243417c \
-                        sha256  ab71e1e5ff665213b7d892ebd0829b3da835538a73409fe11bb006ec5b6b3909
+checksums               md5     6409f5c38840b053d946fb17cc3f2400 \
+                        sha1    5e7b17414533005c4f163bec8be41d023b1f0810 \
+                        rmd160  d1ce73ff32e2e644a5fa02fc5d6b0fdd55d54416 \
+                        sha256  063bf66e163f43b7d7897ac14efe1e80ed094d4016afe1181fe2285e3797bed3 \
+                        size    369664
 
-depends_lib             bin:java:kaffe
+java.version            1.7+
+java.fallback           openjdk8
```

Comment:
The fallback is a MacPorts-provided Java that is installed when an existing Java (version 7 or higher) installation isn't already present on the system. I have specified an LTS OpenJDK version for the fallback, however since I do not know if JSch is compatible with Java 11, I have specified OpenJDK 8 to be less disruptive. However OpenJDK 11 (the more recent LTS version) would be preferred if this port is known to work fine it.

Reply:
so, are you testing whether the port works with OpenJDK 11 or is this ready to merge?

Reply:
I don't think I can test this anytime soon. If the maintainer has nothing to say, then I guess this is ready to merge.